### ISSUE TITLE
Creating users from console not working

### ISSUE THREAD

**Issue by faenir** [2amigos/yii2-usuario#48]
Can't create user from console. Get error "Please fix following errors:" without any error messages.
```
$ yii user/create admin@test.dev admin
Please fix following errors:
--> empty line here <--
```
Probably because I override model classes (only table names changed).
> public static function tableName()

**Comment by tonydspaniard** [comment]
@faenir Thank you, I'll see what is happening asap.

**Comment by faenir** [comment]
$ yii user/create admin@test.dev admin password

> Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: 
> Da\User\Command\CreateController::getModelClass()'
> in yiisoft\yii2\base\Component.php:290

**Comment by tonydspaniard** [comment]
@faenir can you please provide the configuration of the module? Everything works on my end.

**Comment by faenir** [comment]
My config:
```php
'components' => [
    'user' => [
        'class' => yii\web\User::class,
        'identityClass' => Da\User\Model\User::class,
    ],
    'authManager' => [
        'class' => Da\User\Component\AuthDbManagerComponent::class,
    ],
],
```

```php
'modules' => [
    'user' => [
        'class' => Da\User\Module::class,
        'classMap' => [
            'User' => common\models\User::class,
            'SocialNetworkAccount' => common\models\SocialAccount::class,
            'Profile' => common\models\Profile::class,
            'Token' => common\models\Token::class,
        ],
    ],
],
```
common\models\User.php
```php
<?php

namespace common\models;

use Da\User\Model\User as BaseUser;

class User extends BaseUser
{
    public static function tableName()
    {
        return '{{%user}}';
    }
}
```
common\models\Profile.php
```php
<?php

namespace common\models;

use Da\User\Model\Profile as BaseProfile;

class Profile extends BaseProfile
{
    public static function tableName()
    {
        return '{{%user_profile}}';
    }
}
```
common\models\SocialAccount.php
```php
<?php

namespace common\models;

use Da\User\Model\SocialNetworkAccount as BaseSocialAccount;

class SocialAccount extends BaseSocialAccount
{
    public static function tableName()
    {
        return '{{%user_social_account}}';
    }
}
```
common\models\Token.php
```php
<?php

namespace common\models;

use Da\User\Model\Token as BaseToken;

class Token extends BaseToken
{
    public static function tableName()
    {
        return '{{%user_token}}';
    }
}
```
```
$ yii user/create admin@test.dev admin
Please fix following errors:
--> empty line here <--
```

After add `var_dump` into `Da\User\Service\UserCreateService::run()`
```php
} catch (Exception $e) {
    $transaction->rollBack();
    var_dump($e->getMessage());
```
get this error in console:
```
$ yii user/create admin@test.dev admin
string(75) "Process could not be started [The system cannot find the path specified.]"
Please fix following errors:
--> empty line here <--
```

**Comment by faenir** [comment]
The error was in the mailer component.
```php
'mailer' => [
    'class' => yii\swiftmailer\Mailer::class,
],
```
`'useFileTransport' => true` - solved the problem.

But, in my opinion, it's wrong that does not return any error messages.

**Comment by tonydspaniard** [comment]
re #79

**Comment by tonydspaniard** [comment]
This should have been fixed with #79
